lib/dal/src/schema/variant.rs

```diff
@@ -301,6 +303,36 @@ impl SchemaVariant {
         Ok(schema_variant_ids)
     }
 
+    pub async fn get_default_for_schema(
+        ctx: &DalContext,
+        schema_id: SchemaId,
+    ) -> SchemaVariantResult<Self> {
+        // The first pass here is that we will just return the first schema variant found for a schema
+        // We will need to add a default edge at some point in the future - that means when we import
+        // a schema variant (if it doesn't already exist) then it gets an edge for default
+        // If we are creating a new version of a schema variant then we will be able to mark it as default
```

Comment:
We discussed this and this is good for now. We'll need to change it sooner rather than later though. Probably you get an additional default edge when creating the first `SchemaVariant` that you can change.